requirements/prod.txt

```diff
@@ -6,4 +6,4 @@ pandas==2.1.4
 ndjson>=0.3.1
 py-multiformats-cid>=0.4.4
 boto3>=1.26.12
-SQLAlchemy<2.0
+SQLAlchemy<3.0
```

Comment:
I think you should be more explicit here and use >=2.0 since that what it seems like you're aiming for.